### ISSUE TITLE
Fix some weird problems

### ISSUE THREAD

**Issue by MysteryBlokHed** [MysteryBlokHed/databind#36]
One main issue to track things that are very broken.

- [ ] Fix output folder
      When running `databind` with no arguments, the output folder is
      created relative to the directory the command was run is as opposed to
      the root of the project.
- [ ] Fix problems with tagging
      Tags of folders that are *not* in subfolders have a preceding slash before
      the function name (eg. `namespace:/funcname` instead of `namespace:funcname`)
- [ ] Fix while loops breaking
       `.databind` files seem to stop being transpiled after a while loop, and the while loop
       functions are not generated. I'm not sure what's causing this.

**Comment by MysteryBlokHed** [comment]
It looks like some of these problems aren't actually Linux-exclusive, I just found them while I was testing on Linux. I'm not sure how a lot of them occurred.

**Comment by MysteryBlokHed** [comment]
I'm not sure how most of these came up. There were never any problems testing on Windows, and it seems like some of the things added in older pull requests (eg. proper output folder running `databind` with no args in #28) have just stopped functioning.

**Comment by MysteryBlokHed** [comment]
The issue about Windows while loops using full paths as the subfolder prefix was actually just me having a malformed file structure. It might be worth adding some sort of `create-namespace` subfunction to `databind create` to stop people from making the same mistake I did 🙃

Previously, the issue was:
```md
- [ ] Fix while loops breaking
    - [ ] On Windows
          On Windows, the while loops function calls seem to insert the entire path to where
          an mcfunction would be generated?
```

**Comment by MysteryBlokHed** [comment]
On Linux, it looks like the tokenizer still thinks that it's in a while loop after the `:endwhile`. I'm trying to fix this now.

**Comment by MysteryBlokHed** [comment]
It looks like none of the problems were actually Linux-related, I just found them while testing on Linux.

**Comment by MysteryBlokHed** [comment]
Closed in #37
